utils/cdk-nag-scan/bin/cdk-nag-scan.ts

```diff
@@ -5,7 +5,22 @@ import { CdkNagScanStack } from '../lib/cdk-nag-scan-stack';
 import { AwsSolutionsChecks } from 'cdk-nag';
 
 const app = new cdk.App();
-new CdkNagScanStack(app, 'CdkNagScanStack');
+
+const templateFileName = app.node.tryGetContext('fileName');
+if (!templateFileName) {
+  throw new Error('fileName is required');
+}
+
```

Comment:
A comment here indicating that this is creating a stack name that is the last up to 128 characters of the path-name to the CloudFormation template that is found, slugifying non-whitespace characters to `-`, might be appreciated.